packages/ui-extensions/docs/surfaces/point-of-sale/reference/components/TextArea.doc.ts

```diff
@@ -22,9 +22,15 @@ const data: ReferenceEntityTemplateSchema = {
       title: 'Basic TextArea',
       tabs: [
         {
+          title: 'tsx',
```

Comment:
You should be able to use the helper function to get these consistent. https://github.com/Shopify/ui-extensions/blob/my-TextField-doc-update/packages/ui-extensions/docs/surfaces/point-of-sale/reference/helpers/generateCodeBlock.ts